pkg/blockstorage/vmware/vmware.go

```diff
@@ -52,8 +55,10 @@ var (
 
 // FcdProvider provides blockstorage.Provider
 type FcdProvider struct {
-	Gom *vslm.GlobalObjectManager
-	Cns *cns.Client
+	Gom        *vslm.GlobalObjectManager
+	Cns        *cns.Client
+	TagManager tagManager
```

Comment:
Expose the tagging service client too - the `FcdProvider` type is used in CBT which could need it over time.

Hide your internal interface variable - it is really tacky anyway to expose a variable to an internal interface.